src/Chunk.cpp

```diff
@@ -139,7 +139,8 @@ cChunk::~cChunk()
 	{
 		if (!(*itr)->IsPlayer())
 		{
-			(*itr)->Destroy(false);
+			// Scheduling a normal destruction is neither possible (Since this chunk will be gone till the schedule occurs) nor necessary.
+			(*itr)->DestroyNoScheduling(false);  // No point in broadcasting in an unloading chunk. Chunks unload when no one is nearby.
```

Comment:
Specifically, any future code shuffling may break the assumption that an entity is always within its parent chunk's coordinates when `DoWithChunk` is called. I am not even sure this is always the case in the current code.